odxtools/cli/list.py

```diff
@@ -130,6 +132,14 @@ def add_subparser(subparsers: "argparse._SubParsersAction") -> None:
     )
     _parser_utils.add_pdx_argument(parser)
 
+    parser.add_argument(
+        "--no-strict",
+        action="store_true",
```

Comment:
suggestion: use 
BooleanOptionalAction
see https://stackoverflow.com/a/68837302/1438522

Reply:
as far as I can see this was added in python 3.9, i.e. python 3.8 does not support this? (luckily we're slowly closing in on the EOL of Ubuntu 20.04, when we can raise the minimum python version to 3.10...)